content/en/docs/concepts/overview/working-with-objects/annotations.md

```diff
@@ -17,7 +17,8 @@ objects. Labels can be used to select objects and to find
 collections of objects that satisfy certain conditions. In contrast, annotations
 are not used to identify and select objects. The metadata
 in an annotation can be small or large, structured or unstructured, and can
-include characters not permitted by labels.
+include characters not permitted by labels. It is also possible to use labels with 
+annotations to attach metadata to Kubernetes objects
```

Comment:
1. Missing period for the sentence.
2. What do you mean by "use labels with annotations"?

Reply:
@tengqm Thank you for pointing out the mistake. I have added missing period for the sentence. Moreover, I have made the sentence more clearer in the latest commit. As mentioned in the issue #42524, we can use labels as well as annotations in metadata of same object. Therefore, I added a clearer sentence in the end of the paragraph.